Cargo.toml

```diff
@@ -27,3 +27,6 @@ regex = "1.1.0"
 lazy_static = "1.2.0"
 single = "1.0.0"
 quoted-string = "0.6.0"
+
+[profile.release]
+debug = true
```

Comment:
Is there a way to utilize the [`bench` profile](https://doc.rust-lang.org/cargo/reference/manifest.html#the-profile-sections) instead? Seems like an easy mistake to have a larger than intended binary in the actual release.

Reply:
The idea behind `debug=true` in release builds is the `ICE` diagnostic, which accesses the backtrace to generate an output. Otherwise we'd be presented with a useless bug report in release builds.

Reply:
Oh yes, good point. This just turns out to also helping benchmarking in the same way.